analytics/src/FirebaseAnalytics.cs

```diff
@@ -248,6 +251,59 @@ public static void SetConsent(System.Collections.Generic.IDictionary< ConsentTyp
     FirebaseAnalyticsInternal.SetConsentWithInts(consentSettingsMap);
   }
 
+  /// @brief Sets default event parameters.
+  ///
+  /// These parameters are logged with all events, in addition to the parameters passed to the
+  /// LogEvent() call. They are useful for logging common parameters with all events.
+  /// Default event parameters are overridden by event-specific parameters if the names are the
+  /// same. Default parameters are removed if the dictionary is null or empty.
```

Comment:
Let's change it so that default parameters are only cleared if the dictionary is null. *Individual* parameters will be removed (by the underlying implementation) if the value of a given key is null.